lxc/exec.go

```diff
@@ -45,7 +45,7 @@ func (c *execCmd) usage() string {
 	return i18n.G(
 		`Execute the specified command in a container.
 
-lxc exec [remote:]container [--mode=auto|interactive|non-interactive] [--env EDITOR=/usr/bin/vim]... <command>
+lxc exec [remote:]container [--mode=auto|interactive|non-interactive] [--env EDITOR=/usr/bin/vim]... <command> [-- <args>]
```

Comment:
Both work, but they mean different things. I'd be fine with `[--] <command line>`; but I do think it's worth noting that the -- is optional.